cli/mqtt.py

```diff
@@ -22,7 +26,7 @@ def mqtt_open(config, insecure):
             acct.mqtt_username,
             acct.mqtt_password,
             printer.mqtt_key,
-            ca_certs="examples/ankermake-mqtt.crt",
+            ca_certs=cert_path,
```

Comment:
We can just write `ROOT_DIR / "ssl/ankermake-mqtt.crt"` here, without need for `cert_path`

Reply:
✅